apps/xprof_core/src/xprof_core_vm_info.erl

```diff
@@ -86,3 +89,51 @@ get_all_functions(Mod, ModeCB) ->
 get_modules() ->
     ModsFiles = lists:sort(code:all_loaded()),
     [ Mod || {Mod, _File} <- ModsFiles].
+
+%% @doc Return list of called functions for given mfa tuple.
+-spec get_called_funs(mfa()) -> [mfa()].
+get_called_funs({Mod, Fun, Arity}) ->
+    %% Get file for given module and disassemble it
+    File = code:which(Mod),
+    {beam_file, Mod, _, _, _, Disasm} = beam_disasm:file(File),
```

Comment:
if you `-include_lib("compiler/src/beam_disasm.hrl").` then you can use
`#beam_file{module = Mod, code = Disasm} = ...` here
and `#function{name = Fun, arity = Arity, code = Opcodes}` below.

(BTW we need to have `compiler` included in `xprof_core.app.src` -> `{applications,[...]}`, this ensures that the `compiler` application and the `beam_disasm` module is included in the user's release, so we can call it at runtime. This seems to be an additional dependency but both `lager` and `elixir` depend on it so this should not be a problem)